L1Trigger/L1THGCal/interface/veryfrontend/HGCalVFECompressionImpl.h

```diff
@@ -2,14 +2,28 @@
 #define __L1Trigger_L1THGCal_HGCalVFECompressionImpl_h__
 
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include "DataFormats/ForwardDetId/interface/HGCalDetId.h"
 
 class HGCalVFECompressionImpl
 {
 
  public:
   HGCalVFECompressionImpl(const edm::ParameterSet& conf);
+
+  uint32_t bitLength(uint32_t x);
+  void compress(const std::map<HGCalDetId, uint32_t>&,
+                std::map<HGCalDetId, std::array<uint32_t, 2> >&);
+  uint32_t compressSingle(const uint32_t value);
+  uint32_t decompressSingle(const uint32_t code);
```

Comment:
Actually the `compressSingle` and `decompressSingle` methods should also be `private`

Reply:
I made it public at the moment because I want to test it via standalone C++ code.